src/proto/Nebulas.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package TW.Nebulas.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+    // Recipient's address.
+    string from_address = 1;
+
+    // Chain identifier (256-bit number)
+    bytes chain_id = 2;
+
+    // Nonce (256-bit number)
+    bytes nonce = 3;
+
+    // Gas price (256-bit number)
+    bytes gas_price = 4;
+
+    
+    // Gas limit (256-bit number)
+    bytes gas_limit = 5;
+
+    // Recipient's address.
+    string to_address = 6;
+
+    // Amount to send in wei (256-bit number)
```

Comment:
`wei`(ethereum?) or other unit?

Reply:
1 NAS = 10^18 Wei, It's just like Ethereum; :)